src/sssom/context.py

```diff
@@ -25,108 +22,79 @@
 )
 
 
-def get_jsonld_context():
-    """Get JSON-LD form of sssom_context variable from auto-generated 'internal_context.py' file.
-
-    [Auto generated from sssom.yaml by jsonldcontextgen.py]
-
-    :return: JSON-LD context
-    """
+def _get_jsonld_context():
```

Comment:
This wont be needed anymore after this PR is done.

Reply:
okay great then i'll just delete it. using this is a terrible pain